src/collective/ifttt/interfaces.py

```diff
@@ -1,8 +1,23 @@
 # -*- coding: utf-8 -*-
 """Module where all interfaces, events and exceptions live."""
 
+from collective.ifttt import _
+from plone.autoform import directives
+from zope import schema
+from zope.interface import Interface
 from zope.publisher.interfaces.browser import IDefaultBrowserLayer
 
 
 class ICollectiveIftttLayer(IDefaultBrowserLayer):
     """Marker interface that defines a browser layer."""
+
+
+class IIftttControlPanel(Interface):
+
+    # A field in 'hidden' mode
+
+    directives.mode(secret='hidden')
```

Comment:
@Shriyanshagro Why hidden? Doesn't it prevent it being displayed on control panel form?

Reply:
Actually. That line should do nothing. If the schema would contain field named "secret", it would configure it to be hidden. So that line should be removed.

Reply:
done